common/scala/build.gradle

```diff
@@ -13,9 +13,9 @@ dependencies {
 
     compile 'io.spray:spray-json_2.11:1.3.3'
 
-    compile 'com.typesafe.akka:akka-actor_2.11:2.5.4'
-    compile 'com.typesafe.akka:akka-stream_2.11:2.5.4'
-    compile 'com.typesafe.akka:akka-slf4j_2.11:2.5.4'
+    compile 'com.typesafe.akka:akka-actor_2.11:2.5.6'
+    compile 'com.typesafe.akka:akka-stream_2.11:2.5.6'
```

Comment:
This is useful, since the `FileIO.fromPath` source was very recently upgraded to a more performant one.